workflows/bacterial_genomics/quality_and_contamination_control/quality_and_contamination_control.ga

```diff
@@ -0,0 +1,811 @@
+                },
+                "TagDatasetActionoutput_json": {
+                    "action_arguments": {
+                        "tags": "tooldistillator_results"
+                    },
+                    "action_type": "TagDatasetAction",
+                    "output_name": "output_json"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/tooldistillator/tooldistillator/0.8.5.0+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "65a69de1e294",
+                "name": "tooldistillator",
+                "owner": "iuc",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"log\": false, \"tool_section\": {\"tools\": [{\"__index__\": 0, \"select_tool\": {\"tool_list\": \"fastp\", \"__current_case__\": 5, \"input\": {\"__class__\": \"ConnectedValue\"}, \"trimmed_forward_R1_path\": {\"__class__\": \"ConnectedValue\"}, \"trimmed_reverse_R2_path\": {\"__class__\": \"ConnectedValue\"}, \"html_report_path\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"0.23.2\", \"reference_database_version\": null}}, {\"__index__\": 1, \"select_tool\": {\"tool_list\": \"kraken2\", \"__current_case__\": 8, \"input\": {\"__class__\": \"ConnectedValue\"}, \"seq_classification_file_path\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"2.1.1\", \"reference_database_version\": {\"__class__\": \"ConnectedValue\"}}}, {\"__index__\": 2, \"select_tool\": {\"tool_list\": \"bracken\", \"__current_case__\": 4, \"input\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"2.8\", \"reference_database_version\": {\"__class__\": \"ConnectedValue\"}, \"kraken_report_path\": {\"__class__\": \"ConnectedValue\"}, \"threshold\": null, \"read_len\": \"100\", \"level\": \"S\"}}, {\"__index__\": 3, \"select_tool\": {\"tool_list\": \"recentrifuge\", \"__current_case__\": 12, \"input\": {\"__class__\": \"ConnectedValue\"}, \"rcf_stat_path\": {\"__class__\": \"ConnectedValue\"}, \"rcf_html_path\": {\"__class__\": \"ConnectedValue\"}, \"analysis_software_version\": \"1.12.1\", \"reference_database_version\": {\"__class__\": \"ConnectedValue\"}}}]}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "0.8.5.0+galaxy0",
+            "type": "tool",
+            "uuid": "1b3fe024-07d9-493f-8ea6-b9d41ee0f621",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "tooldistillator_results",
+                    "output_name": "output_json",
+                    "uuid": "cc18f393-547e-4d7f-b836-fedda6aa613f"
+                }
+            ]
+        },
+        "10": {
+            "annotation": "ToolDistillator summarize groups all JSON file into a unique JSON file",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/tooldistillator_summarize/tooldistillator_summarize/0.8.5.0+galaxy0",
```

Comment:
same naive question as in #438 - why is this necessary?

Reply:
ah, got it now at last - it's one tool run of ToolDistillator, but that produces a collection of json files that you are merging at this step.